Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -358,8 +358,8 @@ void ProjectData::parseLinearSolvers(BaseLib::ConfigTree const& config)
         BaseLib::insertIfKeyUniqueElseError(
             _linear_solvers,
             name,
-            MathLib::createLinearSolver<GlobalMatrix, GlobalVector,
-                                        GlobalLinearSolver>(&conf),
+            std::unique_ptr<GlobalLinearSolver>(
+                new GlobalLinearSolver("", &conf)),
```

Comment:
it's a prefix for command line options (coming from PETSc). I just copied these from  MathLib::createLinearSolver